suites/reef/nvmeof/tier-1_nvmeof_sanity.yaml

```diff
@@ -122,7 +122,7 @@ tests:
               count: 1
               size: 10G
             listener_port: 5001
-            allow_host: "*"
+            initiator_node: node10
```

Comment:
A suggestion, make it list, so it would accomodate many host.

Reply:
Keep it as `hosts: []`.

Reply:
Done